packages/itwin/measure-tools/src/api/FormatterUtils.ts

```diff
@@ -7,58 +7,88 @@ import type { Point3d, XAndY } from "@itwin/core-geometry";
 import type { Cartographic } from "@itwin/core-common";
 import { IModelApp, QuantityType } from "@itwin/core-frontend";
 import type { FormatterSpec } from "@itwin/core-quantity";
+import { MeasureTools } from "../MeasureTools";
 
 export class FormatterUtils {
-
   private static removeUnitSuffixes(s: string) {
     s = s.replace(/ m/g, "");
     s = s.replace(/ ft/g, "");
     return s;
   }
 
-  private static formatCoordinatesWithSpec(point: Point3d, spec: FormatterSpec): string {
-    const xStr = FormatterUtils.removeUnitSuffixes(IModelApp.quantityFormatter.formatQuantity(point.x, spec));
-    const yStr = FormatterUtils.removeUnitSuffixes(IModelApp.quantityFormatter.formatQuantity(point.y, spec));
-    const zStr = FormatterUtils.removeUnitSuffixes(IModelApp.quantityFormatter.formatQuantity(point.z, spec));
-
-    return IModelApp.localization.getLocalizedString("MeasureTools:Formatting.xyzPoint", { x: xStr, y: yStr, z: zStr });
+  private static formatCoordinatesWithSpec(
```

Comment:
not necessary, but it seems like you could easily just combine the formatCoordinateXYWithSpec and FormatCoordinatesWithSpec